### PR TITLE
Deleting ASE code

### PR DESCRIPTION
**Pull request by adusumillipraveen**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -3,24 +3,10 @@ provider "azurerm" {
...
-
-  s2sUrl = "http://rpe-service-auth-provider-${local.local_env}.service.${local.local_ase}.internal"
-  fees_register_url = "http://fees-register-api-${local.local_env}.service.${local.local_ase}.internal"
-
-  website_url = "http://${var.product}-api-${local.local_env}.service.${local.local_ase}.internal"
-
-  asp_name = "${var.env == "prod" ? "payment-api-prod" : "${var.core_product}-${var.env}"}"
-
-  sku_size = "${var.env == "prod" || var.env == "sprod" || var.env == "aat" ? "I2" : "I1"}"
+  s2sUrl = "http://rpe-service-auth-provider-${var.env}.service.core-compute-${var.env}.internal"
```
Comment (amitchidrewar1301) [inline infrastructure/main.tf:9]: @adusumillipraveen Why we have added this url , I believe its already present in values.yaml

Comment (adusumillipraveen) [reply infrastructure/main.tf:9]: It is used in here https://github.com/hmcts/ccpay-payment-app/blob/2df7614f4b8b1781edaf24c551401bd709009a43/infrastructure/main.tf#L102 which still needs to be present.

Comment (amitchidrewar1301) [reply infrastructure/main.tf:9]: yes I can see its used in api policy, all good for me

Comment (YasharRahvar) [reply infrastructure/main.tf:9]: This is being used in API gateway I believe

**Pull request merged**